app/src/main/java/com/kickstarter/KSApplication.java

```diff
@@ -86,7 +85,7 @@ public boolean isInUnitTests() {
   }
 
   private void checkForCrashes() {
-    final String appId = this.build.isExternal()
+    final String appId = Build.isExternal()
```

Comment:
it failed because [`checkForCrashes()`](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/KSApplication.java#L42) happens before the [build is injected](https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/KSApplication.java#L61)